androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/currentpatients/CurrentPatientItem.kt

```diff
@@ -0,0 +1,141 @@
+package com.EENX15_22_17.digital_journal.android.ui.currentpatients
+
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.rounded.Info
+import androidx.compose.material.icons.rounded.Notifications
+import androidx.compose.material.icons.rounded.Warning
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.window.Dialog
+import com.EENX15_22_17.digital_journal.android.ui.theme.colorIcon
+import com.EENX15_22_17.digital_journal.android.ui.theme.danger
+import com.EENX15_22_17.digital_journal.android.ui.theme.primaryColor
+
+class CurrentPatientItem {
+
+    @Composable
+    fun PatientCard(
```

Comment:
It's OK to use default values since the PreviewParameter-annotation seems to be buggy. But we should probably have a TODO here so that we remember to fix this when we manage to get the tooling working correctly
```suggestion
    // TODO: Replace default parameter-values with PreviewParameter-annotations when tooling is less buggy.
    fun PatientCard(
```